### ISSUE TITLE
Cannot set pre-warp in openalpr-net dynamically

### ISSUE THREAD

**Issue by dpower-lv** [openalpr/openalpr#331]
PreWarp setter in AlprConfigNet doesn't seem to have any effect on configuration.
Example:
`alpr.Configuration.PreWarp = "planar,800.000000,461.000000,0.000650,0.000800,0.150000,1.000000,1.500000,0.000000,0.000000";`

**Comment by matthill** [comment]
The prewarp value is not loaded dynamically from the config.  We added a new function to the library, that allows dynamically setting the prewarp:
https://github.com/openalpr/openalpr/blob/master/src/openalpr/alpr.h#L144

However, the C# interface has not been extended to support it yet.  If you update the C# binding to support this function call, it should give you the behavior you want.

**Comment by dpower-lv** [comment]
Yeah, that seems to work. What is the intended behaviour of the setter? I could submit a pull request containing neccessary changes.

**Comment by newsages** [comment]
I have ...

https://github.com/openalpr/openalpr/pull/304

Bool Config::updatePrewarp(string configFile, string newprewarp)
 {
CSimpleIniA iniObj;
iniObj.LoadFile(configFile.c_str());
CSimpleIniA* ini = &iniObj;
ini->SetValue("","prewarp", newprewarp.c_str());

 return ini->SaveFile(configFile.c_str());
}

 string Config::getPrewarp(string configFile)
 {
CSimpleIniA iniObj;
 iniObj.LoadFile(configFile.c_str());
CSimpleIniA* ini = &iniObj;
return prewarp = getString(ini, "", "prewarp", "");
 } 

For calibration utils.
